### PR TITLE
Zijian/update sample data

### PR DESCRIPTION
**Pull request by BobbyZhouZijian**
Update sample data to include lessons.

Update dates for usage messages.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeadlineCommand.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
             + "[" + PREFIX_TAG + "TAG] \n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_TITLE + "Do experiment "
-            + PREFIX_DATE_TIME + "28-10-2020 12:00 "
+            + PREFIX_DATE_TIME + "18-11-2020 12:00 "
```
Comment (MarcTzh) [inline src/main/java/seedu/address/logic/commands/DeadlineCommand.java:30]: this date is not in the coming week

File: src/main/java/seedu/address/logic/commands/EditTaskCommand.java [hunk 2]
```diff
@@ -54,7 +54,7 @@
             + "[" + PREFIX_END_TIME + "EVENT END TIME (for event only)] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_DATE_TIME + "02-02-2020 12:00 "
+            + PREFIX_DATE_TIME + "20-11-2020 12:00 "
```
Comment (MarcTzh) [inline src/main/java/seedu/address/logic/commands/EditTaskCommand.java:57]: not in the coming week

**Review by MarcTzh — COMMENTED** [review]
not sure if these edits are intended, once you confirm them i can merge

File: src/main/java/seedu/address/logic/commands/DeadlineCommand.java (hunk 1 above)
Comment (BobbyZhouZijian) [reply src/main/java/seedu/address/logic/commands/DeadlineCommand.java:30]: i think this is correct.

File: src/main/java/seedu/address/logic/commands/EditTaskCommand.java (hunk 2 above)
Comment (BobbyZhouZijian) [reply src/main/java/seedu/address/logic/commands/EditTaskCommand.java:57]: ^

**Review by MarcTzh — APPROVED** [review]
LGTM

**Pull request merged**